### PR TITLE
Bump stylus version to 0.53

### PR DESCRIPTION
**Pull request by Cactucs**

### CODE REVIEW

**Comment by enzy** [comment]
Is it enough to update a version used by gulp-stylus?

**Comment by Cactucs** [comment]
Yes

**Comment by Cactucs** [comment]
I'll have to look at it because https://github.com/stevelacy/gulp-stylus/commit/fd333968d65b6d54359fe168a4359fb23e65b295

**Comment by enzy** [comment]
Why not update the gulp-stylus instead?

**Comment by Cactucs** [comment]
Because at the time of change the new version of gulp-stylus wasn't released

**Comment by Cactucs** [comment]
I have updated the dependency so it requires gulp-stylus which requires stylus 0.53

File: package.json [hunk 1]
```diff
@@ -72,7 +72,7 @@
     "gulp-replace": "^0.5.3",
     "gulp-sass": "^2.0.0",
     "gulp-sourcemaps": "^1.2.8",
-    "gulp-stylus": "^2.0.0",
+    "gulp-stylus": ">=2.1.2",
```
Comment (enzy) [inline package.json]: Please same format as other dependencies: `^2.1.2`. Your `>=` is too vague and can breake anytime.

Comment (Cactucs) [inline package.json]: :white_check_mark: Done

**Comment by enzy** [comment]
:+1:

**Pull request merged**